client/app/screens/Login.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Login from 'react-native-login';
+import Icon from 'react-native-vector-icons/FontAwesome';
```

Comment:
But so does Ionicons which we're using throughout the rest of the app? See https://docs.nativebase.io/COMPONENTS.html#Icon_Default & https://ionicframework.com/docs/ionicons/.